lms/static/certificates/sass/_build.scss

```diff
@@ -1,45 +1,53 @@
 // ------------------------------
```

Comment:
Also, it sucks that Sass interpolation can't handle passing a pathing variable (like, ``$edx-pl-path``) into all of these ``@import`` statements.